### DISCUSSION TITLE
[FEATURE] Duplicate tab shortcut

### DISCUSSION THREAD

**Discussion by cherrynoize** [Floorp-Projects/Floorp#795] [Feature Request]
I did post this is the Reddit but I suppose this is the best place to ask for this. I would love to see an inbuilt shortcut for duplicate tab. Or at least, could I achieve this with custom commands?

**Comment by Telmesomething** [comment]
There's a shortcut available already on the context menu. Simply right click the tab you want to duplicate and you'll see it on the menu.

**Reply by Prodigist** [reply]
Not much of a shortcut, Keyboard Shortcut is preferable. Hopefully they implement Ctrl +Shift + K like Edge

**Comment by breadbeforerice** [comment]
+1

**Comment by rguhr** [comment]
Keyboard shortcut and mouse gesture are nowadays possible

**Reply by cherrynoize** [reply]
Not using Floorp anymore so I can't check this but I'm closing then.
